internal/collections/named.go

```diff
@@ -94,11 +94,11 @@ func (c *NamedCollection) String() string {
 }
 
 func (c *NamedCollection) addName(key string) {
-	for _, n := range c.names {
-		if n == key {
-			return
-		}
-	}
+	// for _, n := range c.names {
+	// 	if n == key {
+	// 		return
+	// 	}
+	// }
 	c.names = append(c.names, key)
```

Comment:
We don't need a `names` field then as it won't be much slower to iterate over the map contents in the names view